WordPress/src/main/java/org/wordpress/android/reader/savedposts/JetpackReaderSavedPostsFlag.kt

```diff
@@ -8,5 +8,5 @@ class JetpackReaderSavedPostsFlag @Inject constructor(
     private val jetpackReaderSavedPostsFeatureConfig: JetpackReaderSavedPostsFeatureConfig,
     private val buildConfigWrapper: BuildConfigWrapper
 ) {
-    fun isEnabled() = jetpackReaderSavedPostsFeatureConfig.isEnabled() && buildConfigWrapper.isJetpackApp
+    fun isEnabled() = buildConfigWrapper.isJetpackApp
```

Comment:
I just wanted to confirm if we really should remove the feature config check here or if this was removed for testing.

Reply:
Sorry, I forgot to list this in the description. :man_facepalming:  This is indeed only for this draft PoC. I've included this "hack" to make testing easier, and we can revert this commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17398/commits/305d390b24bbd82313222272446664a05aeba9e7 when we want to proceed. Then, for further testing, we can locally "revert the revert" for convenience.

Reply:
I've updated the description to mention the hard-coded flags as a prerequisite.